migration-engine/connectors/sql-migration-connector/src/flavour/mssql.rs

```diff
@@ -54,9 +56,9 @@ impl SqlFlavour for MssqlFlavour {
                 applied_steps_count     INT NOT NULL DEFAULT 0,
                 script                  NVARCHAR(MAX) NOT NULL
```

Comment:
heads up: this column is going to be gone after https://github.com/prisma/prisma-engines/pull/1419 is merged (soon)

Reply:
PR is merged, `script` column is gone

Reply:
Took this one out too.